packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -12,7 +10,6 @@
             ></ovh-manager-banner>
             <ovh-manager-hub-carousel
                 class="w-100"
-                items="$ctrl.notifications"
```

Comment:
We should be sure of that because if you remove the items property here, you will fetch `notifications` API to display the carousel, but `notifications` API is also called to initialize notifications badges in navbar + notifications sidebar.